### PR TITLE
DOC-717: Remove examples from core nodes

### PR DESCRIPTION
**Pull request by freakwriter**
- Removed examples or example usage section from core nodes
- Removed now-unused _images files

Question:
_snippets/integrations/builtin/core-nodes/merge/if-merge-branch-execution.md has a screenshot example, but there is also a note to remove once v1 is mature. Should this snippet be removed entirely at this point?

### CODE REVIEW

File: docs/integrations/builtin/core-nodes/n8n-nodes-base.merge.md [hunk 1]
```diff
@@ -110,115 +110,6 @@ The items passed into Input 1 of the Merge node will take precedence. For exampl
 
 --8<-- "_snippets/integrations/builtin/core-nodes/merge/if-merge-branch-execution.md"
 
-
```
Comment (StarfallProjects) [inline docs/integrations/builtin/core-nodes/n8n-nodes-base.merge.md:113]: Keep this example

File: docs/integrations/builtin/core-nodes/n8n-nodes-base.scheduletrigger.md [hunk 2]
```diff
@@ -23,35 +23,12 @@ If a workflow uses the Schedule node as a trigger, make sure that you save and a
 
 Select an interval in **Trigger Interval**. n8n displays options for the selected interval.
 
-### Example
```
Comment (StarfallProjects) [inline docs/integrations/builtin/core-nodes/n8n-nodes-base.scheduletrigger.md:26]: Keep this example

File: docs/integrations/builtin/core-nodes/n8n-nodes-base.splitinbatches.md [hunk 3]
```diff
@@ -26,41 +26,6 @@ For usage examples and templates to help you get started, refer to n8n's [Loop O
 n8n automatically processes incoming items. You may not need the Loop Over Items node in your workflow. To learn more about how n8n handles multiple items, refer to the documentation on [Looping in n8n](/flow-logic/looping/).
 ///
 
-## Example usage: Read RSS feed from two different sources
```
Comment (StarfallProjects) [inline docs/integrations/builtin/core-nodes/n8n-nodes-base.splitinbatches.md:29]: Keep this example

**Review by StarfallProjects — CHANGES_REQUESTED** [review]
Hiya! A few changes to revert please.

**Comment by ko-support** [comment]
@StarfallProjects I opted to revert all the commits and redo based on your feedback, so at this point the removed examples and related images only touch these files:

- executecommand
- rssfeedread
- ssetrigger
- ssh
- xml

**Pull request merged**